docs/providers/aws/examples/hello-world/csharp/README.md

```diff
@@ -27,6 +27,18 @@ zip -r ./deploy-package.zip ./*
 popd
```

Comment:
Didn't want to rip it without asking, but with the build scripts in place, maybe this block adds noise?